connectors/src/lib/error.ts

```diff
@@ -22,3 +22,10 @@ export function errorFromAny(e: any): Error {
     stack: e.stack || "No stack trace",
   };
 }
+
+export type WorkflowErrorType = "unhandled_internal_activity_error";
+
+export type WorkflowError = {
+  type: string;
```

Comment:
I presume you wanted to type it `WorkflowErrorType` ?

Reply:
yes.